### PR TITLE
Fix #2405 : Ktlint maxline profileprogress test package

### LINKED ISSUE #2405: Resolve ktlint max line in profileprogress test package
**Describe the bug**
We are preventing the 100 char length at some places like test case fun name. 
`// ktlint-disable max-line-length`

**Expected behavior**
We need to remove this prevention and fix the function name under 100 char length.

### PR DESCRIPTION
**Pull request by varunvjha**
## Explanation

Fixes #2405  : Removed ' // ktlint-disable max-line-length ' and fixed the functions name under 100 char length.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 1]
```diff
@@ -201,7 +201,7 @@ class ProfileProgressFragmentTest {
   }
 
   @Test
-  fun testProfileProgressFragment_openProfilePictureEditDialog_configurationChange_dialogIsStillOpen() { // ktlint-disable max-line-length
+  fun testProfileProg_openProfilePictureEditDialog_configChange_dialogIsStillOpen() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]: please keep the prefix unchanged `testProfileProgressFragment`

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]:
```kotlin
fun testProfileProgressFragment_openProfilePictureEditDialog_configChange_dialogIsStillOpen()
```

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 2]
```diff
@@ -290,7 +290,7 @@ class ProfileProgressFragmentTest {
   }
 
   @Test
-  fun testProfileProgressFragmentNoProgress_recyclerViewItem0_checkOngoingTopicsCount_countIsZero() { // ktlint-disable max-line-length
+  fun testProfileProgNoProg_rVItem0_checkOngoingTopicsCount_countIsZero() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]:
```kotlin
fun testProfileProgressFragment_noProgress_ongoingTopicCountIsZero()
```

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 3]
```diff
@@ -303,7 +303,7 @@ class ProfileProgressFragmentTest {
   }
 
   @Test
-  fun testProfileProgressFragmentWithProgress_recyclerViewItem0_checkOngoingTopicsCount_countIsTwo() { // ktlint-disable max-line-length */
+  fun testProfileProgWithProg_rVItem0_checkOngoingTopicsCount_countIsTwo() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]:
```kotlin
fun testProfileProgressFragment_twoPartialStoryProgress_ongoingTopicCountIsTwo()
```

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 4]
```diff
@@ -325,7 +325,7 @@ class ProfileProgressFragmentTest {
   }
 
   @Test
-  fun testProfileProgressFragmentWithProgress_change_configuration_recyclerViewItem0_checkOngoingTopicsCount_countIsTwo() { // ktlint-disable max-line-length
+  fun testProfileProgWithProg_change_config_rVItem0_checkOngoingTopicsCount_countIsTwo() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]:
```kotlin
fun testProfileProgressFragment_configChange_twoPartialStoryProgress_ongoingTopicCountIsTwo()
```

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 5]
```diff
@@ -349,7 +349,7 @@ class ProfileProgressFragmentTest {
   }
 
   @Test
-  fun testProfileProgressFragmentNoProgress_recyclerViewItem0_checkOngoingTopicsString_descriptionIsCorrect() { // ktlint-disable max-line-length
+  fun testProfileProgNoProg_rVItem0_checkOngoingTopicsString_descriptionIsCorrect() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]:
```kotlin
fun testProfileProgressFragment_noProgress_ongoingTopicDescriptionIsCorrect()
```

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks @varunvjha 
Please apply the similar approach to other test as well and keep a consistency in naming all the fucntion names. 

Also, please update the test name in file `ProfilePictureActivityTest`

```kotlin
fun testProfilePictureActivity_userImageIsDisplayed()
```

**Comment by anandwana001** [comment]
@varunvjha please reply to all the comment above with done or if anything needs to discuss regarding that comment.

**Comment by varunvjha** [comment]
> Thanks @varunvjha 
> Please apply the similar approach to other test as well and keep a consistency in naming all the fucntion names. 
> 
> Also, please update the test name in file `ProfilePictureActivityTest`
> 
> ```kotlin
> fun testProfilePictureActivity_userImageIsDisplayed()
> ```

Done

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt (hunk 1 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]: > please keep the prefix unchanged `testProfileProgressFragment`

Done

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt (hunk 2 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt (hunk 3 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt (hunk 4 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]: Done

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt (hunk 5 above)
Comment (varunvjha) [reply app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]: Done

**Comment by varunvjha** [comment]
> @varunvjha please reply to all the comment above with done or if anything needs to discuss regarding that comment.

Done. PTAL.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Keep consistency with the `testProfileProgressFragment`

at line 186 - testProfileProgressFragment_profilePictureEditDialogIsDisplayed

at line - 251 - testProfileProgressFragment_imageSelectAvatar_configChange_checkGalleryIntent

at line 498 - testProfileProgressFragment_configChange_firstStory_storyNameIsCorrect

at line 515 - testProfileProgressFragment_firstStory_storyNameIsCorrect

at line 536 - testProfileProgressFragment_firstStory_topicNameIsCorrect

at line 557 - testProfileProgressFragment_clickFirstStory_opensTopicActivity

at line 581 - testProfileProgressFragment_clickViewAll_opensRecentlyPlayedActivity

at line 602 - testProfileProgressFragment_noProgress_topicCountIsNotClickable

at line 618 - testProfileProgressFragment_noProgress_storyCountIsNotClickable

at line 634 - testProfileProgressFragment_configChange_noProgress_storyCountIsNotClickable

at line 652 - testProfileProgressFragment_clickTopicCount_opensOngoingTopicListActivity

at line 684 - testProfileProgressFragment_clickStoryCount_opensCompletedStoryListActivity

**Review by anandwana001 — CHANGES_REQUESTED** [review]
testProfileProgressFragment_configurationChange_checkProfileName_profileNameIsCorrect
use -> `configChange`

testProfileProgressFragment_changeConfig_checkOngoingTopicDescriptionIsCorrect
use -> `configChange`

testProfileProgress_twoPartialStoryProgress_checkCompletedStoriesDescriptionIsCorrect
use -> `testProfileProgressFragment` 
is it doesn't comes in 100 char let me know

**Comment by varunvjha** [comment]
Made requested changes. PTAL @anandwana001

**Comment by anandwana001** [comment]
```diff
- testProfileProgressFragment_twoPartialStoryProgress_checkCompletedStoriesDescriptionIsCorrect
+ testProfileProgressFragment_twoPartialStoryProgress_completedStoriesDescriptionIsCorrect

- testProfileProgressFragment_twoPartialStoryProgress_checkOngoingTopicDescriptionIsCorrect
+ testProfileProgressFragment_twoPartialStoryProgress_ongoingTopicDescriptionIsCorrect

- testProfileProgressFragment_configChange_checkOngoingTopicDescriptionIsCorrect
+ testProfileProgressFragment_configChange_ongoingTopicDescriptionIsCorrect

- testProfileProgressFragment_noProgress_checkCompletedStoriesCountIsZero
+ testProfileProgressFragment_noProgress_completedStoriesCountIsZero

- testProfileProgressFragment_twoPartialStoryProgress_checkCompletedStoriesCountIsTwo
+ testProfileProgressFragment_twoPartialStoryProgress_completedStoriesCountIsTwo

- testProfileProgressFragment_noProgress_completedStoriesDescriptionIsCorrect
+ testProfileProgressFragment_noProgress_completedStoriesDescriptionIsCorrect
```

File: app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt [hunk 6]
```diff
@@ -469,7 +469,7 @@ class ProfileProgressFragmentTest {
   }
 
   @Test
-  fun testProfileProgressFragmentWithProgress_recyclerViewItem0_checkCompletedStoriesString_descriptionIsCorrect() { // ktlint-disable max-line-length
+  fun testProfileProgressFragment_twoPartialStoryProgress_checkCompletedStoriesDescriptionIsCorrect() {
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfileProgressFragmentTest.kt]:
```diff
- testProfileProgressFragment_twoPartialStoryProgress_checkCompletedStoriesDescriptionIsCorrect
+ testProfileProgressFragment_twoPartialStoryProgress_completedStoriesDescriptionIsCorrect

- testProfileProgressFragment_twoPartialStoryProgress_checkOngoingTopicDescriptionIsCorrect
+ testProfileProgressFragment_twoPartialStoryProgress_ongoingTopicDescriptionIsCorrect

- testProfileProgressFragment_configChange_checkOngoingTopicDescriptionIsCorrect
+ testProfileProgressFragment_configChange_ongoingTopicDescriptionIsCorrect

- testProfileProgressFragment_noProgress_checkCompletedStoriesCountIsZero
+ testProfileProgressFragment_noProgress_completedStoriesCountIsZero

- testProfileProgressFragment_twoPartialStoryProgress_checkCompletedStoriesCountIsTwo
+ testProfileProgressFragment_twoPartialStoryProgress_completedStoriesCountIsTwo

- testProfileProgressFragment_noProgress_completedStoriesDescriptionIsCorrect
+ testProfileProgressFragment_noProgress_completedStoriesDescriptionIsCorrect
```

**Review by anandwana001 — CHANGES_REQUESTED** [review]

**Review by anandwana001 — APPROVED** [review]
LGTM

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**